packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/Unbranded-TypeSpec.tsp

```diff
@@ -366,3 +366,31 @@ op stillConvenient(): void;
 @head
 @convenientAPI(true)
 op headAsBoolean(@path id: string): void;
+
+@route("/audio")
```

Comment:
lets convert these to unit tests only, cadl ranch will cover these for generated shape.